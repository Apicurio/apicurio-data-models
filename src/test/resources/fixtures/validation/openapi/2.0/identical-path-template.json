{
    "swagger":"2.0",
    "info":{
        "description":"This is a sample server Petstore server via JSON.",
        "version":"1.0.0",
        "title":"Swagger Petstore (JSON)"
    },
    "paths":{
        "/pathstest25": {
            "get": {
                "description": "pathstest25 - non parameterized paths that vary only in trailing slash (/pathstest25) - invalid",
                "responses": {
                    "default": {
                        "description": "..."
                    }
                }
            }
        },
        "/pathstest25/": {
            "get": {
                "description": "pathstest25 - non parameterized paths that vary only in trailing slash (/pathstest25/) - invalid",
                "responses": {
                    "default": {
                        "description": "..."
                    }
                }
            }
        },
        "/pathstest26/{var}": {
            "get": {
                "description": "pathstest26 - parameterized paths that vary only in trailing slash (/pathstest26/{var}) - invalid",
                "responses": {
                    "default": {
                        "description": "..."
                    }
                }
            }
        },
        "/pathstest26/{var}/": {
            "get": {
                "description": "pathstest26 - parameterized paths that vary only in trailing slash (/pathstest26/{var}/) - invalid",
                "responses": {
                    "default": {
                        "description": "..."
                    }
                }
            }
        },
        "/pathstest27/{var1}": {
            "get": {
                "description": "pathstest27 - parameterized paths that vary only in formal parameter names (/pathstest27/{var1}) - See: OAS 3 - Path Template Matching - invalid",
                "responses": {
                    "default": {
                        "description": "..."
                    }
                }
            }
        },
        "/pathstest27/{var2}/": {
            "get": {
                "description": "pathstest27 - parameterized paths that vary only in formal parameter names (/pathstest27/{var2}) - See: OAS 3 - Path Template Matching - invalid",
                "responses": {
                    "default": {
                        "description": "..."
                    }
                }
            }
        }
    }
}