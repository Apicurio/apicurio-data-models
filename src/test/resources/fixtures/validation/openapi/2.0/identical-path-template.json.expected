[PATH-009] |medium| {/paths[/pathstest25]->null} :: Path template "/pathstest25" is semantically identical to at least one other path.
[PATH-009] |medium| {/paths[/pathstest25/]->null} :: Path template "/pathstest25/" is semantically identical to at least one other path.
[PATH-009] |medium| {/paths[/pathstest26/{var}]->null} :: Path template "/pathstest26/{var}" is semantically identical to at least one other path.
[PATH-009] |medium| {/paths[/pathstest26/{var}/]->null} :: Path template "/pathstest26/{var}/" is semantically identical to at least one other path.
[PATH-009] |medium| {/paths[/pathstest27/{var1}]->null} :: Path template "/pathstest27/{var1}" is semantically identical to at least one other path.
[PATH-009] |medium| {/paths[/pathstest27/{var2}/]->null} :: Path template "/pathstest27/{var2}/" is semantically identical to at least one other path.